pkg/auth/service.go

```diff
@@ -364,7 +364,8 @@ func (a *APIAuthService) UpdateUserFriendlyName(ctx context.Context, userID stri
 func (a *APIAuthService) CreateGroup(ctx context.Context, group *model.Group) (*model.Group, error) {
 	ctx = httputil.SetClientTrace(ctx, "api_auth")
 	resp, err := a.apiClient.CreateGroupWithResponse(ctx, CreateGroupJSONRequestBody{
-		Id: group.DisplayName,
+		Id:          group.DisplayName,
```

Comment:
I'd like to see some unit tests that check things like:
1. Persistence- groups can be set, and get successfully from the service
2. Validation- check for valid group description in terms of maximum length, legal characters, etc.

Reply:
Thanks, added.